src/routes/profile/[address].svelte

```diff
@@ -0,0 +1,115 @@
+<script context="module" lang="ts">
+  /* eslint-disable */
+  /** @type {import('./[slug]').Load} */
+  export async function load({ params, fetch }) {
+    const createdWorkstreams = await workstreamsStore.getWorkstreams(
+      { createdBy: params.address },
+      fetch
+    );
+    const assignedWorkstreams = await workstreamsStore.getWorkstreams(
+      { assignedTo: params.address },
+      fetch
+    );
+
+    return {
+      // status: createdWorkstreams.ok || assignedWorkstreams.ok ? 200 : 500,
+      props: {
+        createdWorkstreams: createdWorkstreams.ok && createdWorkstreams.data,
+        assignedWorkstreams: assignedWorkstreams.ok && assignedWorkstreams.data,
+        address: params.address
+      }
+    };
+  }
+  /* eslint-enable */
+</script>
+
+<script lang="ts">
+  import { goto } from '$app/navigation';
+  import { workstreamsStore } from '$lib/stores/workstreams/workstreams';
+  import type { Workstream } from '$lib/stores/workstreams/types';
+
+  import UserBig from '$components/UserBig.svelte';
+  import Section from '$lib/components/Section.svelte';
+  import ExploreCard from '$components/ExploreCard.svelte';
+  import EmptyState from '$components/EmptyState.svelte';
+
+  export let createdWorkstreams: Workstream[] = [];
+  export let assignedWorkstreams: Workstream[] = [];
+  export let address: string;
+</script>
+
+<svelte:head>
+  <title>Workstreams · Profile</title>
+</svelte:head>
+
+<div class="container">
+  <UserBig {address} />
+  <div class="overview">
+    {#if !assignedWorkstreams && !createdWorkstreams}
+      <EmptyState
+        emoji="👀"
+        headerText="Nothing to see here"
+        text="This address has no associated workstreams"
+        primaryActionText="Go explore"
+        on:primaryAction={() => goto(`/`)}
+      />
+    {:else}
+      {#if assignedWorkstreams.length > 0}
```

Comment:
This will fail if `assignedWorkstreams` is `undefined`.

```suggestion
      {#if assignedWorkstreams?.length > 0}
```

Reply:
https://github.com/radicle-dev/workstreams-app/pull/95/commits/351a00f973b3f697d7691b9665fba007fc85b899